src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -159,7 +159,7 @@ class AuthScreens extends React.Component {
         const modalScreenOptions = {
             headerShown: false,
             cardStyle: getNavigationModalCardStyle(this.props.isSmallScreenWidth),
-            cardStyleInterpolator: modalCardStyleInterpolator,
+            cardStyleInterpolator: (...props) => modalCardStyleInterpolator(this.props.isSmallScreenWidth, ...props),
```

Comment:
Why `(...props)`? it looks like there is only one object argument passed to `cardStyleInterpolator`?

Reply:
ok. Will update.